doc/userguide/rules/flow-keywords.rst

```diff
@@ -403,4 +404,4 @@ The number of packets can be matched exactly, or compared using the _op_ setting
 
 Signature example::
 
- alert ip any any -> any any (msg:"Flow has less than 2000 bytes"; flow.bytes_toserver:<2000; sid:1;)
\ No newline at end of file
+ alert ip any any -> any any (msg:"Flow has less than 2000 bytes"; flow.bytes_toserver:<2000; sid:1;)
```

Comment:
Are you just adding an empty line here ?

Reply:
No changes from my end, honestly don't know what causes this highlighting. If I actually try to add a new empty line after this, git complains and won't allow me to commit.